lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -729,6 +729,101 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		buf[ SETTER ]( idx*BYTES_PER_ELEMENT, value, this._isLE );
 	});
 
+	/**
+	* Copies a portion of a typed array to a new typed array.
+ 	* @name subarray
+ 	* @memberof TypedArray.prototype
+ 	* @type {Function}
+ 	* @param {integer} [begin] - start index (inclusive)
+ 	* @param {integer} [end] - end index (exclusive)
+	* @throws {TypeError} `this` must be a typed array instance
+	* @throws {RangeError} first argument must be in the range of `[0, this.length -1]` while second argument must be in the range of `[0, this.length]`
+	* @throws {TypeError} first argument must be integer, and second argument must be integer
+	* @returns {TypedArray} TypedArray array
+	*
+	* @example
+	* var Float64Array = factory('float64');
+	*
+	* var arr = new Float64Array( 'little-endian' , [0.0, 1.0, 2.0, 3.0, 4.0] );
+	*
+	* var out = arr.subarray();
+	* // returns <Float64Array>
+	*
+	* var len = out.length;
+	* // returns 5
+	*
+	* var bool = out.get( 0 );
+	* // returns 0.0
+	*
+	* bool = out.get( len-1 );
+	* // returns 4.0
+	*
+	* out = arr.subarray( 1, -2 );
+	* // returns <Float64Array>
+	*
+	* len = out.length;
+	* // returns 2
+	*
+	* bool = out.get( 0 );
+	* // returns 1.0
+	*
+	* bool = out.get( len-1 );
+	* // returns 2.0
+	*/
+	setReadOnly( TypedArray.prototype, 'subarray', function subarray( begin, end) {
```

Comment:
```suggestion
	setReadOnly( TypedArray.prototype, 'subarray', function subarray( begin, end ) {
```

Reply:
Thanks for pointing my mistakes, i have resolved it now